site/src/components/theme.ts

```diff
@@ -981,6 +981,7 @@ export const theme = createTheme({
       },
       styleOverrides: {
         root: {
+          minWidth: "125px",
```

Comment:
I'm not sure about setting this globally - we might want a smaller input, e.g. for a number. How about passing this through from the schema inputs instead?

Reply:
Alright, have updated the TextInput field instead of making this a global here: https://github.com/blockprotocol/blockprotocol/pull/167/files#diff-309a59f94b4cdf913ac3ca9228a73fd75b2436c0ec3ed0a525abbaf39a3a8736R73